packages/migrations/src/migrate_snapshot.ts

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+import { devConstants, web3Factory } from '@0x/dev-utils';
+import { logUtils } from '@0x/utils';
+import { Provider } from 'ethereum-types';
+
+import { runMigrationsAsync } from './migration';
+
+(async () => {
+    let providerConfigs;
+    let provider: Provider;
+    let txDefaults;
+
+    providerConfigs = { shouldUseInProcessGanache: true, ganacheDatabasePath: '0x_ganache_snapshot' };
+    provider = web3Factory.getRpcProvider(providerConfigs);
+    txDefaults = {
+        from: devConstants.TESTRPC_FIRST_ADDRESS,
+    };
+    await runMigrationsAsync(provider, txDefaults);
+    process.exit(0);
+})().catch(err => {
+    logUtils.log(err);
+    process.exit(1);
+});
```

Comment:
Can we please at instructions to the README about how to generate a snapshot and upload it to S3?